app/models/shipit/github_status.rb

```diff
@@ -0,0 +1,17 @@
+module Shipit
+  module GithubStatus
+    CACHE_KEY = 'github::status'.freeze
+
+    class << self
+      def status
+        Rails.cache.read(CACHE_KEY)
+      end
+
+      def refresh_status
+        Rails.cache.fetch(CACHE_KEY) do
```

Comment:
Ok, sorry if I missed that previously, but actually this won't work. If you don't specify an expiry, the cache will stay pretty much forever. So `refresh` should use `.write`.